.travis.yml

```diff
@@ -1,21 +1,22 @@
 sudo: false
 language: node_js
 node_js:
-  - "iojs"
   - "0.10"
 
+env:
+  - JPM_FX_DEBUG=0
+  - JPM_FX_DEBUG=1
+
 notifications:
   irc: "irc.mozilla.org#jetpack"
 
 before_install:
   - "export DISPLAY=:99.0"
-  - "sh -e /etc/init.d/xvfb start"
   - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16 -extension RANDR"
 
 before_script:
-  - npm install mozilla-download -g
+  - npm install fx-download -g
   - npm install gulp -g
-  - cd ..
-  - mozilla-download --branch nightly -c prerelease --host ftp.mozilla.org firefox
+  - bash bin/fx-download.sh
   - export JPM_FIREFOX_BINARY=$TRAVIS_BUILD_DIR/../firefox/firefox
   - cd $TRAVIS_BUILD_DIR
```

Comment:
export MOZ_QUIET=1
will help keep the debug logs shorter, though it might make more sense to do that in the node-scripts somewhere.